ethcore/private-tx/src/lib.rs

```diff
@@ -146,13 +148,16 @@ pub struct Receipt {
 
 /// Manager of private transactions
 pub struct Provider {
-	encryptor: Arc<Encryptor>,
-	config: ProviderConfig,
+	encryptor: Box<Encryptor>,
```

Comment:
encryptor is never shared, so there is no reason to make wrap it `Arc`

Reply:
It was before :-(